### ISSUE TITLE
Ignore eslint during builds

### ISSUE THREAD

**Issue by alexfinnarn** [CUCentralAdvancement/essential-cu#197]
https://github.com/CUCentralAdvancement/essential-cu/pull/194/checks?check_run_id=2836108116#step:4:49

Next.js apparently added eslint checks during the build? Not sure why that's the default or encouraged, but since linting runs at the CI level, it just adds unnecessary time to the build process.

**Comment by alexfinnarn** [comment]
I had thought that linting was added to this project, but I only added issue templates to all repos. So https://github.com/CUCentralAdvancement/essential-cu/pull/198 is an attempt to find out what the errors are that the build fails on. 

> If you'd like Next.js to dangerously produce production code even when your application has ESLint errors, you can disable the built-in linting step completely.

I still think running eslint is overkill if you do have a CI check, but I gather that Vercel does this because their hosting platform probably doesn't have a robust tool, like GitHub Actions, to run the check before merging code.
